app/Platform/Services/GameSuggestions/GameSuggestionEngine.php

```diff
@@ -26,6 +27,14 @@ public function __construct(
         $this->initializeStrategies($user);
     }
 
+    /**
+     * For testing purposes only.
+     */
+    public function dangerouslyEnableFixedStrategyForTesting(): void
+    {
+        $this->unsafe_useFixedStrategyForTesting = true;
+    }
```

Comment:
I have a feeling that you're going to want to specify an explicit strategy to test. Wouldn't it make more sense to have this function just set `$this->strategies` to a single strategy (possibly passed in)?

Reply:
Yeah, I like that more. Updated in latest.